scripts/build/builders/host.py

```diff
@@ -351,6 +361,19 @@ def __init__(self, root, runner, app: HostApp, board=HostBoard.NATIVE,
             self.extra_gn_options.append('chip_project_config_include_dirs=["//config/python"]')
             self.build_command = 'chip-repl'
 
+        if self.app == HostApp.SIMULATED_APP1:
+            self.extra_gn_options.extend(
+                [
+                    'chip_tests_zap_config="app1"',
+                ]
+            )
+        if self.app == HostApp.SIMULATED_APP2:
+            self.extra_gn_options.extend(
+                [
```

Comment:
using append instead of extend saves 2-3 lines of vertical space. Extend is only useful of you want to append multiple lines at once.

Reply:
Updated. Thank you for the suggestion.